JamesRexMiller4/palette_picker_BE
# **USER USES TEST DATA TO MODEL OFF OF**

### What does this PR do? 

This PR creates a data.js file comprised of users and data datasets that will be used to seed the USERS, FOLDERS, and PALETTES tables. 

### Where should the reviewer start? 
Reviewer can go to the root directory of the project and open the data.js file and review the two datasets within.

### How should this be manually tested? 

Currently, best testing will need to be postponed until Schemas and Migrations are in place before any seeding can occur.  In the meantime, reviewer should examine the data and make sure that all pertinent information is present. If anything is missing (email address?) just give a shout or leave a comment and suggestions will be added accordingly. 

Resolves #5
